src/pages/commits/ActiveCommits/CommitChartLabel.tsx

```diff
@@ -19,13 +20,34 @@ export const CommitChartLabel: React.FC<Props> = ({
   message,
 }) => {
   const createDate = new Date(createTime);
+  const maxChars = 42;
```

Comment:
nit. move this out of the component body since its a static value. It doesn't need to be initialized every render.